src/protocol/abi/json_abi_format.md

```diff
@@ -74,6 +77,17 @@ The ABI of a contract is represented as a JSON object containing the following p
 
 > **Note**: This JSON should be both human-readable and parsable by the tooling around the FuelVM and the Sway programming language. There is a detailed specification for the binary encoding backing this readable descriptor. The [Function Selector Encoding](./fn_selector_encoding.md) section specifies the encoding for the function being selected to be executed and each of the argument types.
 
+### Attributes Semantics
+
+| Attribute name | Attribute arguments               | Semantics |
```

Comment:
Do you mind "prettyfying" this table? I use [this](https://marketplace.visualstudio.com/items?itemName=darkriszty.markdown-table-prettify) VS code extension for that purpose.

Reply:
Sure. Just did that